CSS/styles.css

```diff
@@ -394,3 +435,288 @@ textarea {
 ::placeholder {
   color: #b3bac5;
 }
+
+/* Desktop Size */
+
+@media only screen and (min-width: 1261px) {
+  :root {
+    --Neturals-1: #091e42;
+  }
+
+  body {
+    display: initial;
```

Comment:
- [ ] Kindly make sure your body also has a similar color to the design provided in Figma.